packages/generators/templates/component/{{name}}.stories.{{mdx}}

```diff
@@ -0,0 +1,101 @@
+import { ArgsTable, Canvas, Meta, Story } from "@storybook/addon-docs";
+import { {{name}} } from ".";
+
+# {{ titleCase name }}
+
```

Comment:
We should add a `Meta` in here

`<Meta title="Components/{{name}}" component={{{name}}} />`

Reply:
Ohh good catch